dbt/include/athena/macros/materializations/seeds/helpers.sql

```diff
@@ -34,3 +43,7 @@
 
   {{ return(sql) }}
 {% endmacro %}
+
+{% macro athena__load_csv_rows(model, agate_table) %}
```

Comment:
where is that used? is dbt-core calling it internally? if so might be worth to add a little comment.